src/Microsoft.AspNet.Http.Abstractions/QueryString.cs

```diff
@@ -148,7 +148,7 @@ public static QueryString Create(IEnumerable<KeyValuePair<string, string>> param
         /// </summary>
         /// <param name="parameters"></param>
         /// <returns>The resulting QueryString</returns>
-        public static QueryString Create(IEnumerable<KeyValuePair<string, StringValues>> parameters)
+        public static QueryString Create(IQueryCollection parameters)
```

Comment:
I don't think IQueryCollection->QueryString makes sense, it generally goes the other way. Or do you actually see it used that way somewhere?